lib/airbrake/configuration.rb

```diff
@@ -130,11 +130,8 @@ class Configuration
       },
       lambda { |line| line.gsub(/^\.\//, "") },
       lambda { |line|
-        if defined?(Gem)
-          Gem.path.inject(line) do |l, path|
-            l.gsub(/#{path}/, "[GEM_ROOT]")
-          end
-        end
+        Gem.path.each{ |path| line.sub!(/#{path}/, "[GEM_ROOT]") unless path.to_s.strip.empty? } if defined?(Gem) 
```

Comment:
@developwithvelocity can you help me understand why this could fix that backtrace issue? How can I reproduce this?